src/Nri/Ui/UiIcon/V1.elm

```diff
@@ -13,7 +13,7 @@ module Nri.Ui.UiIcon.V1 exposing
     , edit, pen, highlighter
     , speechBalloon, mail
     , arrowTop, arrowRight, arrowDown, arrowLeft, arrowPointingRight, arrowPointingRightThick, sortArrow, sortArrowDown
-    , checkmark, checkmarkInCircle, x
+    , checkmark, checkmarkInCircle, x, xInCircle
```

Comment:
It's weird to have something called `x` haha but I don't know a better name and it's as descriptive as it gets I guess.